### DISCUSSION TITLE
Does this work on Alma Linux?

### DISCUSSION THREAD

**Discussion by scottiericho** [aws/amazon-cloudwatch-agent#1154] [General]
I have WHM running on a couple of Alma Linux AMIs after CentOS went the way of the warthog. I'd like to monitor my memory usage via Compute Optimiser and need the Cloud Watch Agent running. Will it be fine to install as per AWS' instructions on their own Linux?

**Comment by danburgo** [comment]
AlmaLinux versions 10, 9, and 8 
https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/supported-operating-systems.html
